irr/src/CMakeLists.txt

```diff
@@ -102,6 +116,10 @@ endif()
 
 add_compile_definitions("_IRR_COMPILE_WITH_${DEVICE}_DEVICE_")
 
+if(USE_ANGLE)
+	add_definitions("-D_IRR_COMPILE_WITH_ANGLE_")
```

Comment:
technically this needs a check for iOS, because `SDL_Metal_*` will not work anywhere else.